tools/PI/DevHome.PI/Controls/ExpandedViewControl.xaml

```diff
@@ -146,9 +146,16 @@
                     <TextBlock x:Uid="ExpandedViewAppDiskPerf" Text="{x:Bind viewModel.DiskUsage, Mode=OneWay}"/>
                 </StackPanel>
 
-                <Button HorizontalAlignment="Right" Grid.Column="1" Click="SettingsButton_Click">
-                    <TextBlock Text="&#xe713;" FontFamily="{StaticResource SymbolThemeFontFamily}" FontSize="{StaticResource CaptionTextBlockFontSize}" />
-                </Button>
+                <StackPanel Orientation="Horizontal" Grid.Column="1" HorizontalAlignment="Right">
+                    <StackPanel Orientation="Horizontal" Spacing="10" >
+                        <TextBlock x:Uid="FilterDataToTargetAppToggleLabel" VerticalAlignment="Center"/>
+                        <ToggleSwitch x:Name="ApplyAppFilteringToData" IsOn="{x:Bind viewModel.ApplyAppFiltering, Mode=TwoWay}" Toggled="viewModel.ApplyAppFilteringToData_Toggled"/>

```

Comment:
nit: consider promoting to command using behaviors:
```xaml
<i:Interaction.Behaviors>
    <ic:EventTriggerBehavior EventName="Toggled">
        <ic:InvokeCommandAction Command="{x:Bind viewModel.ApplyAppFilteringCommand}" />
    </ic:EventTriggerBehavior>
</i:Interaction.Behaviors>
```

Reply:
Thanks. I was thrown for a loop when the ToggleSwitch didn't support Commands "natively" like many of the other controls. Hopefully that's something for WinUI3 vNext?